File: WPServer/WPProjectServer/WPProjectServer/Logic/IClientLogic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WPProjectServer.DatabaseObjects;

namespace WPProjectServer.Logic
{
    public interface IClientLogic
    {
        CopyStatus ReturnCopy(string copyCode);

        CopyStatus RentCopy(string copyCode, string userCode);

        bool RegisterUser(User user);
    }
}

File: WPWeb/Library/src/main/java/controllers/IClientLogic.java
package controllers;

import models.*;
/**
 * Created by Szymon on 2015-10-13.
 */
public interface IClientLogic
{
    CopyStatus ReturnCopy(String copyCode) throws Exception;

    CopyStatus RentCopy(String copyCode, String userCode) throws Exception;

    boolean RegisterUser(User user);
}
